tests/Tests.Json.Decode.fs

```diff
@@ -1119,11 +1140,12 @@ Expecting a string but instead got: 12""".Trim())
                 | Error _ -> ()
                 | Ok _ -> failwith "Expected type error for `name` field"
 
-                let expectedMissingField = Ok(None)
-                let actualMissingField =
-                    Decode.fromString (Decode.field "height" (Decode.option Decode.int)) json
+                // TODO: Should this test pass? We should use Decode.optional instead
+                // let expectedMissingField = Ok(None)
+                // let actualMissingField =
+                //     Decode.fromString (Decode.field "height" (Decode.option Decode.int)) json
```

Comment:
There was a little difference in the past between `Decode.field (... Decode.option ...)` and `Decode.optional`

I need to find the discussion about it because one was accepting `null` I think while the other no.